overlay/3-node/patches/pullsecret-sealed-patch.yaml

```diff
@@ -0,0 +1,15 @@
+---
+- op: replace
+  path: /spec/template/metadata/namespace
+  value: 'volante-cars-lab'
+- op: remove
+  path: /metadata/annotations/sealedsecrets.bitnami.com~1cluster-wide
+  value: "true"
+- op: remove
+  # yamllint disable-line rule:line-length
+  path: /spec/template/metadata/annotations/sealedsecrets.bitnami.com~1cluster-wide
+  value: "true"
+- op: replace
```

Comment:
Why you are removing the `cluster-wide` annotation? this allows you to patch the namespace for the secret

Reply:
Just wanted to highlight an example where the SealedSecret has not cluster wide scope and we may care about some extra protection. If we have multiple workload cluster to install we may wanna keep the scope where you can unseal the secret to a fixed namespace. I agree with you though that it is not a default example we wanna have there.